CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+# How To Contribute
+
+## Installation
+
+* `git clone <repository-url>`
+* `cd my-addon`
+* `npm install`
```

Comment:
`npm` -> `yarn`

Reply:
FYI: You can add inline suggestions that can be merged 👍 
https://help.github.com/articles/incorporating-feedback-in-your-pull-request/#applying-a-suggested-change

Reply:
Nice! I'll have a look at that feature.